### DISCUSSION TITLE
Pansharpening

### DISCUSSION THREAD

**Discussion by FelipeSBarros** [semiautomaticgit/SemiAutomaticClassificationPlugin#340] [Q&A]
Dear Luca @semiautomaticgit , I am about to start a class about SCP using the newest version. I could see some changes and a lot of improvents. Congratulations for your work. Thanks for all involved personal.
I saw that the pansharpening process is not anymore in the preprocessing tools. Is there a way to run a pansharpening using SCP, even if I need to use the python script terminal?
If it is posible, would be possible to share some code snippet, so I can at least show my students about the pansharpenning process?
Thanks in advance
Felipe

**Comment by semiautomaticgit** [comment]
Dear @FelipeSBarros , thank you very much, I'm glad that you appreciate the new version.
Unfortunately pansharpening is still not available, I had to remove non essential part of the software to speed up the refactoring.
I'll add it in the future, but I can't say exactly a date.
